plugins/catalog-import/api-report.md

```diff
@@ -102,7 +102,7 @@ export class CatalogImportClient implements CatalogImportApi {
   constructor(options: {
     discoveryApi: DiscoveryApi;
     scmAuthApi: ScmAuthApi;
-    identityApi: IdentityApi;
+    fetchApi: FetchApi;
```

Comment:
@benjdlambert I assume in this PR as well you would like to keep the old api as deprecated?